Garcia-Julz/mine-your-business
# Jg model 1

# Description
Added the modules to migrate to the db.
Closes #[Enter issue #]
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-model-1`
4. ensure that db.sqlite3 shows the tables from the modes created by running your server and making sure things don't blow up with errors.
# Checklist:
